app/uk/gov/hmrc/platformstatusfrontend/controllers/StatusController.scala

```diff
@@ -42,17 +42,19 @@ class StatusController @Inject()(appConfig: AppConfig, mcc: MessagesControllerCo
     val iteration2Future = statusChecker.iteration2Status()
     val iteration3Future = statusChecker.iteration3Status()
     val iteration4Future = statusChecker.iteration4Status()
+    val iteration5Future = statusChecker.iteration5Status()
 
     for {
       iter2 <- iteration2Future
       iter3 <- iteration3Future
       iter4 <- iteration4Future
+      iter5 <- iteration5Future
 
       statusMap = List(statusChecker.iteration1Status(),
```

Comment:
wrap in Future to get syntactic consistency?
Make parallel requests explicit with `.mapN` or something?
(At first glance I thought it was doing 2-5 in parallel, and then 1)
statusMap isn't a map, rename statuses?

Reply:
Good points - will address in subsequent PR.